Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -391,7 +391,7 @@ final class PurchasesOrchestrator {
                 ))
 
                 DispatchQueue.main.async {
-                    completion(nil, nil, error, false)
+                    completion(nil, nil, ErrorUtils.purchasesError(withUntypedError: error).asPublicError, false)
```

Comment:
This is needed because of untyped `throws` in Swift, which doesn't allow us to guarantee type safety here.
However this is pretty well covered in tests.